mne/evoked.py

```diff
@@ -751,7 +751,7 @@ def copy(self):
 
     def __add__(self, evoked):
         """Add evoked taking into account number of epochs"""
-        out = merge_evoked([self, evoked])
```

Comment:
It wasn't in the docs anywhere and it wasn't in the main module, so I
assumed deprecation was unnecessary.